documentation/quarkus/guide-tkit.asciidoc

```diff
@@ -0,0 +1,192 @@
+:toc: macro
+toc::[]
+
+= tkit extensions
+
+1000kit (tkit) is a set of extensions and plugins for Quarkus.
+It was developed internally by Capgemini, so the source code of the extensions is not publicly available.
```

Comment:
all our quarkus libs and extensions have publicly available source code(there was issue with gitlab project visibility, fixed now, the group was public already) and are pushed to Maven OSS https://gitlab.com/1000kit/libs/quarkus